Makefile

```diff
@@ -162,7 +168,16 @@ help:
 	@echo "         make TARGET WORKDIR=/tmp "
 	@echo ""
 	@echo "- REPO_URL"
-	@echo "use it to change the where rootfs base packages will be downloaded"
+	@echo "Use it to change the where rootfs base packages will be downloaded."
+	@echo ""
+	@echo "- IMG_SIZE"
```

Comment:
You get bonus points if this list of variables is sorted alphabetically! :smile:

Reply:
sorted! and removed ugly echo's in help target.